README.md

```diff
@@ -46,50 +48,48 @@ You can find the stylesheet file in app/assets/stylesheets/home.css
+4. After every search is complete, you should add what was searched to the search results sections. Each entry in that section should also list the number of search results returned for that search. You should allow the user to click on that item to re-execute the search. You don't need to persist the history. 
 
-5. Write a quick two or three paragraph summary about how you implemented things and why? What tradeoffs did you make? What are some other choices that you could have made?
+5. Write a two or three paragraph summary about how you implemented things things, and why. What are some other choices that you could have made? What tradeoffs did you make? 
 
 ## Tips
 
-1. Feel free to completely rewrite the provided HTML in app/views/home/index.html if you would do something differently.
+1. Feel free to completely rewrite the provided HTML in app/views/home/index.html.erb if you would do something differently.
 
-2. Feel free to completely rewrite the CSS that is provided in app/assets/stylesheets/home.css.
+2. Feel free to completely rewrite the CSS that is provided in app/assets/stylesheets/home.css.scss.
 
 3. Don't mess with app/views/layouts/application.html.erb or you might end up deep in the bowels of Rails trying to troubleshoot
 
-4. Feel free to add any additional javascript or css files if you want, but know that you might have to fight with Rails alittle to get them working.
+4. Feel free to add any additional javascript or css files if you want. You can add them to /app/assets/javascripts/ and /app/assets/stylesheets. Rails can often have issues with adding new files, for css/javascipt. Don't try to fight rails, we won't count all your code in two files against you.
 
 ## Evaluation
 
-It's always important to know how you are going to be evaluated.  What we are going to look for are pretty straightforward.
+What we are going to look for are pretty straightforward.
 
 1. Does the layout match the mock as closely as possible? Is the CSS well organized?
 
 2. Does the Nile application work the way it should based on the Requirements section?
 
 3. Is the code well organized and does it follow traditional javascript coding conventions?
 
-4. Bells and Whistles. If there is time, what were you able to add. For example, the fonts in the mock are located here: https://fonts.google.com/specimen/Noto+Sans and http://fontawesome.io/assets/font-awesome-4.7.0.zip.
+4. Bells and Whistles. If there is time, how can you improve on the bookstore? For example, the fonts in the mock are located here: https://fonts.google.com/specimen/Noto+Sans and http://fontawesome.io/assets/font-awesome-4.7.0.zip. 
 
-5. Your summary statement. Do your choices make sense? Do you explain them well? Are we left with a lot of open questions?
+5. Your summary statement. Are your choices sensible, and well explained?
```

Comment:
"well explained" is an appositive for choices. Commas go before an appositive. https://www.scribendi.com/advice/what_are_appositives.en.html